TestAppTypescript/package.json

```diff
@@ -13,11 +13,11 @@
   },
   "dependencies": {
     "@react-native-community/async-storage": "^1.5.0",
-    "appcenter": "file:appcenter-2.0.0.tgz",
-    "appcenter-analytics": "file:appcenter-analytics-2.0.0.tgz",
-    "appcenter-crashes": "file:appcenter-crashes-2.0.0.tgz",
-    "appcenter-link-scripts": "file:appcenter-link-scripts-2.0.0.tgz",
-    "appcenter-push": "file:appcenter-push-2.0.0.tgz",
+    "appcenter": "file:appcenter-2.2.0.tgz",
+    "appcenter-analytics": "file:appcenter-analytics-2.2.0.tgz",
```

Comment:
This SDK does not have the new native APIs for hasReceivedMemoryWarning, need to use incremental builds.